app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -251,6 +251,7 @@ def interview_row(application_choice)
     def visa_details_row(application_choice)
       return if (immigration_right_to_work?(application_choice) ||
                 application_predates_visa_sponsorship_information?(application_choice)) ||
+                course_can_sponsor_visa?(application_choice) ||
                 provider_can_sponsor_visa?(application_choice)
```

Comment:
I think we can drop the `provider_can_sponsor_visa?` check now, and rely entirely on the course setting?

```suggestion
                course_can_sponsor_visa?(application_choice)
```

Reply:
Ok, I was in doubt if I should use as failsafe mechanism but happy to remove.

Reply:
Done